src/unittest/python/tracker_tests.py

```diff
@@ -71,3 +71,19 @@ def test_point_not_in_area(self):
         T.wait_for_fixation(self.mock_area, 0.2)
 
         self.assertEqual(len(self.mock_timer_instance.clear.mock_calls), 2)
+
+
+class TestCollectFrames(TestCase):
+
+    @mock.patch('vyu.tracker.image2position')
+    def test_collect_frames_puts_image_position_to_queue(self, mock_i2pos):
+        mock_reader = mock.MagicMock()
+        mock_queue = mock.Mock()
+        mock_reader.__iter__.return_value = ['frame1', 'frame2']
+
+        tracker.collect_frames(mock_reader, mock_queue)
+
+        self.assertEqual(mock_i2pos.call_count, 2)
+        mock_i2pos.assert_has_calls([mock.call('frame1'),
+                                     mock.call('frame2')])
```

Comment:
Should add the `any_order=False` condition to the `assert_has_calls`

Reply:
That's the default.